IFC4x3/Constants/r/RAILWAYCOMMUNICATIONTERMINAL_1C$9BNNcj9mx0W2dT2QsBb.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RAILWAYCOMMUNICATIONTERMINAL_1C9BNNcj9mx0W2dT2QsBb" Name="RAILWAYCOMMUNICATIONTERMINAL" UniqueId="4cfc92d7-5e6b-49c3-b020-0a77426b62e5">
+	<Documentation>A railway communication terminal is an audio communication device that usually installed along the railway in order to be used by general public or railway agents for communication. It may specifically be used to make calls to emergency services in tunnels.</Documentation>
```

Comment:
Should be generalized to something like .COMMUNICATIONTERMINAL. Seems to already be present:
https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC3/HTML/link/ifcaudiovisualappliancetypeenum.htm

I would think that the domain experts can figure out what it is from the dataset context. 😄

Reply:
I agree with this suggestion.